posts/freeing-our-maintainers.md

```diff
@@ -0,0 +1,56 @@
+
+## Postfix: Why our OSS model is broken
+
+Our historic model has been built around the OSS model. It’s relied on volunteers who have come into Exercism, done great work on building tracks and then been given maintainer privileges, in which they can then accept contributions from our wider user base to improve them.
+
+While this seems great on paper, it has some significant problems. At the foremost of these is that the people who add the most magic to Exercism end up not having any time to code or create Exercism because their time gets spent responding to community contributions. This is almost never why the maintainers got involved in Exercism in the first place, and not work they enjoy. It’s a little like someone who loves developing being “promoted” to be a team-leader, where they manage people instead of coding. It might seem like a nice promotion at the time, but often it turns out that people don’t enjoy being managers half as much as they enjoy coding.
+
+It also relies on the assumption that the sum of wider community contributions is greater than the individual contribution that a given maintainer could otherwise make. But in Exercism, that’s almost never the case. Exercism is complex, and education is hard, and together they make contributing to Exercism a complex and hard thing to build. There’s loads to learn and understand about both how Exercism works technically and its approach to education, and so this means that most first contributions are people finding their way. This means their initial contributions are relatively small but also that they nearly always need lots of work in reviewing and tweaking. This is time-consuming work for maintainers. In fact, the total time spent in reviewing (as well as the context-shifting required) means the maintainer generally puts more effort into reviewing the PR than if they’d just made it themselves. There are, of course, some exceptions to this, but it’s true in 99% of cases. And frequently this is even more painful to the maintainer as the problem the PR solves was not near the top of their priority list, meaning the things that they know are really essential don’t get done as a result.
+
+Finally, the OSS model relies on contributors starting off small and eventually becoming knowledgeable and regular enough that they can become maintainers. In OSS projects like software libraries, this works relatively well (e.g. someone is using a library in production and keeps adding improvements to it, to the point they eventually have as much knowledge as the original creator). However, for Exercism, that’s just not happened. Despite merging PRs from thousands of contributors in the last 12 months, only a tiny handful have gone on to be regular contributors.ven fewer have become maintainers. This is again mainly due to the complexity of Exercism, but also because it’s not a contained piece of software, where this model traditionally work.
```

Comment:
```suggestion
Finally, the OSS model relies on contributors starting off small and eventually becoming knowledgeable and regular enough that they can become maintainers. In OSS projects like software libraries, this works relatively well (e.g. someone is using a library in production and keeps adding improvements to it, to the point they eventually have as much knowledge as the original creator). However, for Exercism, that’s just not happened. Despite merging PRs from thousands of contributors in the last 12 months, only a tiny handful have gone on to be regular contributors and even fewer have become maintainers. This is again mainly due to the complexity of Exercism, but also because it’s not a contained piece of software, where this model traditionally work.
```